src/test/mocktracer_implemenation.ts

```diff
@@ -0,0 +1,58 @@
+
+import { expect } from 'chai';
+import {initGlobalTracer, MockTracer } from '../index';
+import { MockSpan } from '../mock_tracer';
+
+export function mockTracerimplementationTests(createTracer = () => new MockTracer()): void {
```

Comment:
why define `createTracer` as argument?

Reply:
This seems to be a copy and paste issue. It could be directly in the test case as well.